recipes/transformers/meta.yaml

```diff
@@ -0,0 +1,60 @@
+{% set name = "transformers" %} 
+{% set version = "2.1.1" %}
+
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: "https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz"
+  sha256: "bc0aee900929ee51ce1ad13e4c00b6b7900177e522ec4b393a2aab10fa02b7bd"
+
+build:
+  noarch: python
+  number: 0
+  entry_points:
+    - transformers=transformers.__main__:main
+  script: "{{ PYTHON }} -m pip install . --no-deps --ignore-installed -vv "
+
+requirements:
+  host:
+    - boto3
```

Comment:
You probably don't need all of these, `python` and `pip` is likely enough for the `host` requirements.